README.md

```diff
@@ -158,8 +158,15 @@ final ListenableFuture<A> firstSuccessful = FuturesExtra.select(futures);
 
 #### Success/Failure callbacks
 
-If you are only interested in either successful or failed results of a future,
-you can use these callbacks:
+You can attach callbacks that are run depending on the results of a future:
+```java
+final ListenableFuture<A> future = getFuture();
+FuturesExtra.addCallback(future, a -> System.out.println(a), e -> e.printStackTrace());
```

Comment:
this example could also use method references: System.out::println and Throwable::printStackTrace